api/src/main/resources/config/application.properties

```diff
@@ -1,12 +1,3 @@
-#logging.level.org.springframework=DEBUG
-
-# This property was added to suppress ElasticSearch RestClient warnings after upgrading
-# ElasticSearch to 7.16.2 for log4j vulnerabilities. We would expect this property to be
-# removed after upgrading to spring-data-elasticsearch 4.2.x
-# https://docs.spring.io/spring-data/elasticsearch/docs/current/reference/html/#preface.requirements
-logging.level.org.elasticsearch.client.RestClient=DEBUG
```

Comment:
I think we could keep this line disabled instead.

Reply:
oops prod. ok!

Reply:
It is moved to the application-dev.properties file, and deleted from here.